using-win32/07-2DRotation/OGL_native.cpp

```diff
@@ -216,6 +216,7 @@ void display(void)
 
 	//to clear all pixels
 	glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+	//Reset model view matrix

```

Comment:
@jeevan-gaikwad  Add `glMatrixMode(GL_MODELVIEW);` line before you call `LoadIdentity`

Reply:
Done